how-to/join_kubeflow_ecosystem.md

```diff
@@ -0,0 +1,110 @@
+# Project Donation Application
+
+Please see the [proposals/new-project-join-process.md](Documentation) to
+better understand the full process for submitting a new project.
+In short, copy this Application Template and populate the document.
+--!>
+
+## CNCF checklist
+- [ ] All project metadata and resources are vendor-neutral
+- [ ] Governance structure
+- [ ] Contributing guides
+- [ ] Overlap with existing Kubeflow projects
+- [ ] Public list of adopters.
+
+
+## Background information
+
+1. Submitter Name
+    -
+
+1. Submitter’s relationship to project / title
+    -
+
+1. Project Name
+    -
+
+1. Why is this project is valuable to the Kubeflow Community?
+    -
+
+1. Why is it beneficial for this project to be a part of the Kubeflow Community?
+    - 
+
+1. List of integrations with Kubeflow Core components (and potential)
+    -
+
+1. Short Description / Functionality
+    -
+
+1. Website
+    -
+
+1. GitHub repository
+    -
+
+1. 1st Release date
+    -
+
+1. Project Meeting Times
+    -
+
+1. Meeting Notes
+    -
+
+1. Installation Documentation
+    -
+
+1. Project Documentation
+    -
+
+1. Security Profile (CVE scanning)
```

Comment:
Podsecuritystandards restricted should be mentioned as well

Reply:
Updated 👍